lib/std/fs/File.zig

```diff
@@ -1617,14 +1617,23 @@ pub const LockError = error{
 /// A process may hold only one type of lock (shared or exclusive) on
 /// a file. When a process terminates in any way, the lock is released.
 ///
-/// Assumes the file is unlocked.
+/// Assumes the file is unlocked, or in the case of Lock.none, locked.
```

Comment:
It is not obvious why you have made this change to the API. Can you explain this in the PR writeup and commit message?

Reply:
this was already the case with the posix API, the documentation and windows API just didn't match it. see this line https://github.com/ziglang/zig/blob/58c50c5a9eb0f326a050c85f7dd50e836ff87f55/lib/std/fs/File.zig#L1737 on master branch.

Reply:
I updated the PR write-up, I'm not sure how to update a commit message, but I'll do that as soon as I figure it out.

Reply:
I updated the commit message